README.md

```diff
@@ -16,3 +17,61 @@ To install dependent modules, double click on ```install_dependencies.py```. Alt
 Please note that you might have to change the LABRADHOST environmental variable after installation. For instructions on how to do this, look up on CampbellGroup wiki.
 
 To install the ok module, you need to go the software folder on the Campbell server and install the corresponding package.
+
+# Conventions
+New code should...  
+* Be Python 2 & 3 compatible  
+* Follow pep8  
+* Have default config files in the config directory, with local users specifying configuration settings via an external "config" directory at the  same level as this repository.    
+* Minimize functional code in guis, i.e. have the GUIs import   functions/objects that perform calculations whenever reasonable.  
+* Test code for all new (and old) but GUIs is highly encouraged.    
+
+
+# Contributing
```

Comment:
we should probably mention the following is adopted from Martinis group procedure